app.py

```diff
@@ -38,8 +39,14 @@ def get_tweets():
     client = Elasticsearch(hosts=[params['es_endpoint']])
     tweets = es_client.get_tweets(client)
 
-    xs = []
-    ys = []
+
+    import dateutil
+
+    id_strs = []
+    created_ats = []
+    full_texts = []
+    subjs = []
```

Comment:
use more descriptive variable names. Is this subjects? and below polarities?

Reply:
Resolved in next commit!